5_databases/week2/hwkpart3task.sql

```diff
@@ -0,0 +1,29 @@
+-- hwk part 3 week 2 ---
+
+Get all the tasks assigned to users whose email ends in @spotify.com
+
+SELECT * FROM task
+JOIN user ON task.user_id = user.id
+WHERE user.email LIKE '%@spotify.com';
+
+Get all the tasks for 'Donald Duck' with status 'Not started'
+
+SELECT * 
+FROM task
+WHERE user_id = (SELECT id FROM user WHERE name = 'Donald Duck')
```

Comment:
You do not have to change anything, but I want you to briefly consider how you would do this with joins instead? 😄